pkg/operator/podidentity/podidentitywebhook_controller.go

```diff
@@ -325,8 +350,8 @@ func (r *staticResourceReconciler) ReconcileResources(ctx context.Context) error
 		// Don't deploy the PDB to single replica topology clusters
 		r.logger.Debugf("not deploying PodDisruptionBudget to single replica topology")
 	} else {
-		// "v4.1.0/aws-pod-identity-webhook/poddisruptionbudget.yaml"
-		requestedPDB := resourceread.ReadPodDisruptionBudgetV1OrDie(v410_00_assets.MustAsset("v4.1.0/aws-pod-identity-webhook/poddisruptionbudget.yaml"))
+		// "v4.1.0/common/poddisruptionbudget.yaml"
+		requestedPDB := resourceread.ReadPodDisruptionBudgetV1OrDie(v410_00_assets.MustAsset(pdb))
```

Comment:
nit: if `pdb` is used once and only once, the indirection of having a constant is not worth it in my opinion

Reply:
It is used twice so i think it is worth const.